tests/suites/test_suite_rsa.function

```diff
@@ -24,8 +24,8 @@ void mbedtls_rsa_pkcs1_sign( data_t * message_str, int padding_mode,
                              char * input_N, int radix_E, char * input_E,
                              data_t * result_hex_str, int result )
 {
-    unsigned char hash_result[1000];
-    unsigned char output[1000];
+    unsigned char hash_result[MBEDTLS_MD_MAX_SIZE];
+    unsigned char output[256];
```

Comment:
Is 256 always enough? How do we know?

Reply:
I ran the tests on PC and printed the values. 
If we write a new trest, that uses more than 256, then it will not fit. Perhaps we should consider dynamic allocations, but I am less favor on this